### DISCUSSION TITLE
side views camera: labeling

### DISCUSSION THREAD

**Discussion by cosimogonnelli** [talmolab/sleap#822] [Help!]
Hi All,

I was wondering if you have any advice for the labeling of mice when using side views cameras. 
I feel like videos in this case are less reliable then videos recorded using top-down cameras. 
You can end up having frames where a lot of labels are not visible at all. For example, you can have a mouse that is facing away from the camera making his snout impossible to label.

How can you create a skeleton that is more reliable in these type of vides? 

Thanks!

**Comment by talmo** [comment]
Hi @cosimogonnelli,

This is a bit tricky -- side views are hard with all the occlusions. Bottom-up models where you can label a very central node like mid-spine can work well by having edges going _from_ the central node _to_ all other nodes -- and no other edges. In other words, only the centermost (and most visible) node should be a source node for all edges. This reduces dependency on intermediate nodes by having a "shallower" skeleton tree.

**Reply by cosimogonnelli** [reply]
Hi @Talmo,

Thanks for your help! it is exactly what I am trying to do but it is still very hard for the reasons you mentioned even though it seems doing better than using a more `classic` skeleton where I try to follow the anatomy of the animal.

**Comment by olinesn** [comment]
Hi @cosimogonnelli,

I'm a staff scientist in the Falkner lab at Princeton, and we're working toward improving our mouse side model as well. It's definitely in the "needs improvement" box at the moment. We're feeding labelled frames from as many different varied contexts as possible, which only works if everyone is using the same skeleton. I'm happy to share tips, etc. and if you're getting started on a mouse side model, perhaps you would want to use our skeleton if you don't have too many hours in yet?

Here's a short clip that should give you an idea.

Best,
Stefan

https://user-images.githubusercontent.com/69451171/178851784-22809338-f432-4328-ad34-df7448e08dd8.mp4

**Reply by cosimogonnelli** [reply]
Hi @olinesn,

Thanks for reaching out and for your help!  I would be happy to have a chat with you. How would you prefer to connect? 

Best,
Cosimo

**Reply by olinesn** [reply]
Happy to chat, you can email me at soline@princeton.edu. Can zoom as early as 4p today.

**Comment by cosimogonnelli** [comment]
Hi @olinesn,

I tried to send you an email a couple of time. I got both times an automatic reply without body text. Since I am not sure you actually got my emails, I am trying to contact you here since I have no other ways to connect. Hope to hear from you and to be able to connect. 
My email is: cosimo.gonnnelli@mso.umt.edu 


Sorry to send this type of message here since is not the appropriate platform...

**Reply by olinesn** [reply]
Sorry that was my away message while I'm traveling, I'll look for the email and respond to you there.
